modules/runner/src/main/java/org/apache/ignite/internal/app/IgniteImpl.java

```diff
@@ -944,4 +944,14 @@ public BiPredicate<String, NetworkMessage> dropMessagesPredicate() {
     public void stopDroppingMessages() {
         ((DefaultMessagingService) clusterSvc.messagingService()).stopDroppingMessages();
     }
+
+    /**
+     * Gets a hybrid clock specific of the node.
```

Comment:
"specif of the node", sounds a bit unnatural from my point of view.
I'd rather use "Returns the node's hybrid clock." or similar.